test/ViewModel/Factory/PictureBuilderFactoryTest.php

```diff
@@ -170,6 +170,37 @@ public function it_creates_a_builder_for_images_that_does_not_stretch()
         );
     }
 
+    /**
+     * @test
+     */
+    public function it_creates_a_builder_for_images_that_can_stretch_if_there_is_a_height()
+    {
+        $factory = new PictureBuilderFactory();
+
+        $image = $this->createImage([
+            'uri' => 'https://example.com/image',
+            'alt' => 'alt',
+            'size' => [
+                'width' => 200,
+                'height' => 100,
+            ],
+            'source' => [
+                'mediaType' => 'image/jpeg',
+                'uri' => 'https://example.com/image.jpg',
+                'filename' => 'Image.jpg',
+            ],
+        ]);
+        $builder = $factory->forImage($image, 400, 200);
```

Comment:
just pointing out 200% is the current configured limit for stretching https://github.com/elifesciences/iiif-formula/blob/master/salt/iiif/config/etc-loris2-loris2.conf#L39

Reply:
Yep, https://github.com/elifesciences/journal/blob/develop/src/Helper/CreatesIiifUri.php#L16-L18 bakes in that limit.